src/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars :first-child {
+  margin-left: 0;
+}
+
+.stars__star {
+  background-image: url("/images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+}
+
+.stars__star--active {
+  background-image: url("/images/star-active.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+}
```

Comment:
Two things about this piece of code:

1. You are setting all of these styles in a selector above. Get rid of `.stars_star--active`(all stars should have one class - "stars__star")  class and create a different selector which changes only `background-image: url("/images/star-active.svg");`

2. Instead of using class with modificator, try to determine which star needs to have different background image with help of :nth-child and stars--N
You can check the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child)